src/apps/applets/commands/applet_ema.py

```diff
@@ -127,3 +189,180 @@ async def export_flows():
+                row.end_date = cast(datetime.date, row.end_date)
+                row.start_date = cast(datetime.date, row.start_date)
+                last_weekday = FRIDAY_WEEKDAY if not row.is_crossday_event else SATURDAY_WEEKDAY
+                if schedule_date.weekday() <= last_weekday and schedule_date <= row.end_date:
+                    filtered.append(row)
+            case PeriodicityType.WEEKLY:
+                row.end_date = cast(datetime.date, row.end_date)
+                row.start_date = cast(datetime.date, row.start_date)
+                scheduled_weekday = row.start_date.weekday()
+                following_weekday = scheduled_weekday
+                if row.is_crossday_event:
+                    # For crossday event need to check following day as well
+                    following_weekday += 1
+                    # If was SUNDAY start new week
+                    if following_weekday > SUNDAY_WEEKDAY:
+                        following_weekday = 0
+                    # Increase end_date for crossday event only in case if weekdays of start and end days are equal.
+                    # The same logic on frontend for schedule.
+                    if row.start_date.weekday() == row.end_date.weekday():
+                        row.end_date += datetime.timedelta(days=1)
+                if (
+                    schedule_date.weekday() == scheduled_weekday or schedule_date.weekday() == following_weekday
+                ) and schedule_date <= row.end_date:
+                    filtered.append(row)
+            case PeriodicityType.MONTHLY:
+                row.end_date = cast(datetime.date, row.end_date)
+                row.start_date = cast(datetime.date, row.start_date)
+                scheduled_monthday = row.start_date.day
+                following_monthday = scheduled_monthday
+                if row.is_crossday_event:
+                    following_monthday += 1
+                    if is_last_day_of_month(row.start_date):
+                        following_monthday = 1
+                    # Increase end_date only if start_date equal end_date
+                    if (
+                        is_last_day_of_month(row.start_date)
+                        and is_last_day_of_month(row.end_date)
+                        or row.start_date.day == row.end_date.day
+                    ):
+                        row.end_date += datetime.timedelta(days=1)
+                if (
+                    schedule_date.day == scheduled_monthday
+                    or schedule_date.day == following_monthday
+                    or (is_last_day_of_month(schedule_date) and row.start_date)
+                ) and schedule_date <= row.end_date:
+                    filtered.append(row)
+    return filtered
+
+
+@app.command(short_help="Export daily user flow schedule events to csv")
+@coro
+async def generate_user_flow_schedule(run_date: datetime.datetime = typer.Argument(..., help="run date")):
```

Comment:
Please make it optional. Date param should point to the expected export date if it set, or prev date if it not set